docs/src/development/transfer-dedicated.md

```diff
@@ -1,149 +1,135 @@
 
-```bash
-rsync -avzP pub/static/ <USERNAME>@<CLUSTER_NAME>.ent.platform.sh:pub/static/
-```
+{{< /note >}}
 
-Replace `pub/static` with the path to your files on system, such as `web/sites/default/files/`.
-Note that rsync is very picky about trailing `/` characters.
-Consult the rsync documentation for more that can be done with that command.
+To synchronize your database:
 
-## Synchronizing the database from development to staging/production
+1. Export the Development database on your local computer:
 
-The database can be copied directly from the development environment to staging or production,
-but doing so requires noting the appropriate credentials first on both systems.
+   ```bash
+   platform db:dump --project=<PROJECT_ID> --environment=<DEVELOPMENT_ENVIRONMENT> --file=dump.sql
+   ```
 
-First, log in to the production environment over SSH:
+   For more options, see [database export](../add-services/_index.md#exporting-data).
 
-```bash
-ssh <USERNAME>@<CLUSTER_NAME>.ent.platform.sh
-```
+2. Import the Development database dump file into the remote Staging/Production database:
+
+   ```bash
+   platform sql --project=<PROJECT_ID> --environment=<PRODUCTION_ENVIRONMENT> < dump.sql
+   ```
+
+## Get SSH access
 
-Once there, you can look up database credentials by running:
+You can access a specific environment of your project via SSH [using the CLI](../development/cli/_index.md):
 
 ```bash
-echo $PLATFORM_RELATIONSHIPS | base64 --decode | json_pp
+platform ssh --project=<PROJECT_ID> --environment=<ENVIRONMENT>
 ```
 
-Which should give a JSON output containing something like this:
+For more granular connection options, [see SSH connection details](../development/ssh/_index.md#get-ssh-connection-details).
 
-```json
-{
-   "database" : [
-      {
-         "password" : "abc123",
-         "username" : "projectname",
-         "path" : "projectname",
-         "port" : "3306",
-         "scheme" : "mysql",
-         "host" : "127.0.0.1",
-         "query" : {
-            "is_master" : true,
-            "compression" : true
-         }
-      }
-   ]
-}
-```
+## Get the database details
 
-The part we want is the host, user, password, and the "path", which is the database name.
-Ignore the rest.
+For advanced used cases, you might want to get the database details manually.
 
-Now, in a separate terminal log in to the development instance using `platform ssh`.
-Run the same `echo` command as above to get the credentials for the database on the development instance.
-(The JSON will be slightly different but again we're only interested in the user, password, host, and "path"/database name).
+To do so:
 
-With the credentials from both databases,
-we can construct a command that will export data from the development server
-and write it directly to the Dedicated cluster's server.
+1. Access the specific environment you need with `platform ssh --project=<PROJECT_ID> --environment=<ENVIRONMENT>`. See more [about SSH](#get-ssh-access).
```

Comment:
:hammer_and_wrench: 
```suggestion
1. Access the specific environment you need by running `platform ssh --project=<PROJECT_ID> --environment=<ENVIRONMENT>`.
```
The link doesn't add additional information and I don't think it would be so relevant at this point in any case.